packages/docusaurus-theme-classic/src/theme/Blog/Components/Author/index.tsx

```diff
@@ -83,7 +83,7 @@ export default function BlogAuthor({
                 <AuthorName name={name} as={as} />
               </MaybeLink>
             )}
-            {count && <AuthorBlogPostCount count={count} />}
+            {count !== undefined && <AuthorBlogPostCount count={count} />}
```

Comment:
What appearance is intended? If zeros should appear in a badge wouldn't it be sufficient to just render the `AuthorBlogPostCount` no matter what?
```suggestion
            <AuthorBlogPostCount count={count} />
```
Otherwise if it shouldn't show a badge at all if there are no posts, then converting `count` to a bool would be the better alternative:
```suggestion
            {Boolean(count) !== undefined && <AuthorBlogPostCount count={count} />}
```

Reply:
Any number should appear on the badge, however count can be undefined because sometimes we use the component without rendering the count badge (like on a blog post)